<launch>
    <!-- <include file="$(find iiwa_gazebo)/launch/iiwa_gazebo.launch">
    </include> -->
     <!-- Select the robot name -->
    <arg name="robot_name" default="iiwa"/>
    <arg name="model" default="14" />
    <arg name="controller" default="TorqueController"/>


    <include file="$(find iiwa_driver)/launch/iiwa_bringup.launch">
        <arg name="robot_name" value="$(arg robot_name)"/>
        <arg name="model" value="$(arg model)"/>
        <arg name="controller" value="$(arg controller)" />
    </include>

    <group ns="/$(arg robot_name)">
        <node name="passive_null_tracking" pkg="iiwa_toolkit" type="passive_null_track" output="screen">
        </node>

        <rosparam file="$(find iiwa_toolkit)/config/passive_null_track_params.yaml" command="load"/>
    </group>

    <!-- when real use, add the force sensor part here -->

</launch>
